controllers/api/v1/datas.controller.js

```diff
@@ -120,14 +122,20 @@ function ServicesController(options) {
         fields.forEach(function (field) {
           whitelistedFields.push({"name": "data." + field.nameNormalized, "key": field.type });
         });
-        if (typeof res._request !== 'undefined' && res._request.params !== 'undefined');
+        if (typeof res._request !== 'undefined' && res._request.params !== 'undefined') {
           query = FilterService.buildMongoQuery(query, res._request.params.filter, 'Data', whitelistedFields);
+        }
         if (query["ERRORS"] !== undefined && query["ERRORS"].length > 0)
           return next({code: 400, messages: query["ERRORS"]});
         var queryOptions = {
-          sort: {updatedAt: -1, createdAt: -1},
           populate: []
         };
+        if (res._request !== undefined && res._request.params !== undefined && res._request.params.sort !== undefined) {
+          queryOptions = SortService.buildMongoQuery(queryOptions, res._request.params.sort, 'Data', whitelistedFields);
+        }
+        else {
+          queryOptions["sort"] = {updatedAt: -1, createdAt: -1};
```

Comment:
this default value could be placed in the service when `res._request.params.sort` is undefined